src/inputType/PanInput.ts

```diff
@@ -338,6 +338,19 @@ export class PanInput implements InputType {
     });
   }
 
+  protected _getVector(properties: number[], direction: boolean[]): number[] {
```

Comment:
I think while `_applyScale` shows what it actually does with its name (at least a part of it), `_getVector` does not reveal what's happening inside.
Why did you change its name like this?

Reply:
It was a function which applies direction and scale to the given offset and velocity.
It originally had the name `getOffset`, but the name was changed while calculating the velocity other than the offset.

Meanwhile, the class that inherits PanInput from view360 [also overrides this](https://github.com/naver/egjs-view360/blob/master/src/YawPitchControl/input/RotationPanInput.ts#L77). It don't just use scale and direction and use other calculations.
Instead of applyScale, I decided to name it getting a vector which meaning both velocity and offset.

We'll be using these names in `RotationPanInput` at view360 as well. Any good suggestions will be welcome.

Reply:
The name `Vector` is too general, I mean, vector of what is not included.
If you want to use the term `vector`, I would suggest `getOffsetVector` or something.

Reply:
The original name getOffset looks better than getOffsetVector.
But I'm little bit concerned about using this name for the velocity calculation as well.

Reply:
I think you can consider what would be better, and choose it yourself.
Just remember to choose self-representative names for it.